lib/datetime_helper/version.rb

```diff
@@ -0,0 +1,5 @@
+module Labs
+  module DatetimeHelper
+    VERSION = "0.0.3"
```

Comment:
No.  It'll be 1.0.0 when its been code-reviewed, and any module / file naming issues have been cleaned up.